flow/scripts/fillcell.tcl

```diff
@@ -1,4 +1,8 @@
-utl::set_metrics_stage "globalroute__{}"
+if {[expr [file exists $::env(REPORTS_DIR)/congestion.rpt] && \
```

Comment:
FYI, It is likely that this step will get moved after drt in the future.

Reply:
Yes... this support is going to require some maintenance as the flow changes, no way around that. Hopefully not too bad and perhaps we can find some nice way.